### PR TITLE
[MIRROR] [MDB IGNORE] Kapulimbs

### PR DESCRIPTION
**Pull request by JollyStationBot**
Original PR: https://github.com/tgstation/tgstation/pull/65523
-----

## About The Pull Request
Ports Kapulimbs 1.0 & 1.1 from Shiptest, while trimming some unrelated code.

- https://github.com/shiptest-ss13/Shiptest/pull/530
- https://github.com/shiptest-ss13/Shiptest/pull/726

Kapulimbs is a bodypart & update icons rewrite I created with a few specific intentions:

1. Simplify the code
2. Allow players to create frankensteins of different species parts
3. Enable non-humanoid species to be created

### How is this achieved?
Bodyparts are now pathed to their specific species, and they can have their own unique life loops, appearances, and anything else a bodypart child should have. A "bodytypes" bitfield is utilized to differentiate important attributes, such as BODYTYPE_ORGANIC and BODYTYPE_HUMANOID.

## Why It's Good For The Game
Better code AND better fun.
image

## Changelog

:cl:
add: Limbgrower limbs are now a sickly green
refactor: Digitigrade legs should now behave consistently.
refactor: Limbs will no longer update appearance as owner is transferred, allowing for multi-species frankensteins
fix: CHANGELING TRANSFORM FUNCTIONS PROPERLY
del: Synths and synth-related mechanics (Synth fugative)
/:cl:

### CODE REVIEW

**Comment by Jolly-66** [comment]
I take a look at GitHub

Ugh

File: jollystation_modules/code/modules/mob/living/carbon/human/species_types/skrell.dm [hunk 1]
```diff
@@ -68,7 +68,7 @@ GLOBAL_LIST_EMPTY(head_tentacles_list)
 	if(istype(human.wear_mask) && (human.wear_mask.flags_inv & HIDEHAIR))
 		. = FALSE
 	var/obj/item/bodypart/head/our_head = human.get_bodypart(BODY_ZONE_HEAD)
-	if(our_head?.status == BODYTYPE_ROBOTIC)
+	if(our_head?.bodytype == BODYTYPE_ROBOTIC)
```
Comment (Kapu1178) [inline jollystation_modules/code/modules/mob/living/carbon/human/species_types/skrell.dm:71]: this is a bitflag not a single value.

`if(ourhead && !IS_ORGANIC_LIMB(ourhead)`

**Review by Kapu1178 — CHANGES_REQUESTED** [review]

**Comment by Jolly-66** [comment]
@prodirus 
Can you update our modular one with whatever you changed 

Thx bby

**Comment by Jolly-66** [comment]
@prodirus 
Skrell organs are broken but thats for another time

Everything works boys lets pack 'er up.

**Pull request merged**